index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="main.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="menu">
+            <h3>John Doe</h3>
```

Comment:
- [ ] As the previous reviewer requested, please use an `a` tag for the logo instead of an `h3` since in most websites the logo is a link that leads to the home page 🌤️